htdocs/themes/xswatch4/language/english/main.php

```diff
@@ -41,7 +41,7 @@
 define('THEME_FORUM_REGISTER', 'Register');
 define('THEME_FORUM_SEARCH', 'Search');
 define('THEME_FORUM_ADVSEARCH', 'Advanced Search');
-define('THEME_FORUM_REPLY', 'Reply');
+//define('THEME_FORUM_REPLY', 'Reply');
```

Comment:
I still don't think it is appropriate for a module to define a `THEME_` prefix constant, but since this persists, it should be consistent with the same issue changes in xbootstrap:
```//define('THEME_FORUM_REPLY', 'Reply');
if (!defined ('THEME_FORUM_REPLY')) {
    define('THEME_FORUM_REPLY', 'Reply');
}```